concepticondata/concepticon.tsv

```diff
@@ -963,7 +963,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 962	WOMAN	Kinship	An adult human member of the sex that produces ova and bears young.	Person/Thing	
 963	BOOK	Speech and language	A collection of leaves of paper, parchment or other material, usually bound or fastened together within covers, containing writing of any type or blank pages for future inscription.	Person/Thing	
 964	WOOL	Clothing and grooming	The soft, curly hair that forms the fleece of sheep, lamas and other animals and which is used to produce clothing.	Person/Thing	
-965	WORLD	The physical world	The Earth with all its inhabitants and all things upon it.	Person/Thing	
+965	WORLD	The physical world	The earth with all its inhabitants and all things upon it.	Person/Thing	
```

Comment:
I would leave Earth capitalized here as it refers to the name of the planet.

Reply:
Ah, I see.